pages/blog/july-2023.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Community Update: July 2023"
+date: 2023-08-05T06:00:00+01:00
+type: Communication
+tags:
+  - Project Status
+cover: /img/posts/2023-summary/blog-banner-july.webp
+authors:
+  - name: Thulisile Sibanda
+    photo: /img/avatars/thulieblack.webp
+    link: https://www.linkedin.com/in/v-thulisile-sibanda/
+    byline: AsyncAPI Community Manager
+excerpt: 'July Community Update'
+featured: true
+---
+
+Earlier this month, we officially kickstarted the AsyncAPI Mentorship Program. After receiving an overwhelming response of over 90+ applications for the program, we inducted [13 mentees into the AsyncAPI mentorship program](https://github.com/dssitachi/community/blob/e204ff252327aff1f8a1c90d98d65466834c6c76/mentorship/asyncapi-mentorship/2023/README.md). We are thrilled to welcome these talented individuals and look forward to seeing them grow and excel in their respective projects. Additionally, we are excited that our budget of $19,500 got approved, which means we can compensate all successful mentees of the program. We are grateful for the support and look forward to a successful program.
+
+
+## AsyncAPI Conf on Tour 2023 (AACoT) 
+
+Tickets for AACoT'23 London event are available for purchase now, and we have some fantastic talks lined up for September 20th that you won't want to miss. [Hurry and get your tickets through Open Collective](https://opencollective.com/asyncapi/events/asyncapi-conference-on-tour-6b3c0aa1/contribute/asyncapi-conference-on-tour-london-61313) because we have limited availability. 
+
+We are also in the process of reviewing talk proposals and will announce our selected speakers soon. 
+
+Be sure to keep an eye out for our August newsletter, as we will be sharing some exciting news there and revealing the next location.
+
+## Spec x Tooling
+The work on the pre-release version of AsyncAPI Studio is now in progress. The implementation will include the support of validating AsyncAPI v3 documents. If you want to follow the development process or contribute, please [check out the ShapeUP dashboard](https://shapeit.app/projects/org/asyncapi/22/cycles/fa1e9a31?issue=I_kwDODou01c5r29cR).
+On the other hand, we have released the initial versions of three Bindings protocols. Our gratitude goes out to [Adam Retter](https://github.com/adamretter) for the [JMS binding](https://github.com/asyncapi/bindings/tree/master/jms), [Richard Coppen](https://github.com/rcoppen) and [Steve Head](https://github.com/SrfHead) for agreeing to maintain it long-term, and [Ian Cooper](https://github.com/iancooper) and [Dec Kolawski](https://github.com/dpwdec) for the [SNS binding](https://github.com/asyncapi/bindings/tree/master/sns) and [SQS binding](https://github.com/asyncapi/bindings/tree/master/sqs) and being maintainers.
```

Comment:
```suggestion

On the other hand, we have released the initial versions of three protocol bindings. Our gratitude goes out to:
- [Adam Retter](https://github.com/adamretter) for contributing the [JMS binding](https://github.com/asyncapi/bindings/tree/master/jms), [Richard Coppen](https://github.com/rcoppen) and [Steve Head](https://github.com/SrfHead) for agreeing to maintain it long-term
- [Ian Cooper](https://github.com/iancooper) and [Dec Kolawski](https://github.com/dpwdec) for contributing the [SNS binding](https://github.com/asyncapi/bindings/tree/master/sns) and [SQS binding](https://github.com/asyncapi/bindings/tree/master/sqs) agreeing to maintain it long-term
```

kinda better to have it as list
also added "enter" in first line so it renders in article as separate paragraph